src/main/java/com/targomo/client/api/quality/criterion/MobilityCriterionDefinition.java

```diff
@@ -14,6 +15,7 @@
 @SuperBuilder(toBuilder = true)
 @ToString(callSuper = true)
 @JsonDeserialize(builder = MobilityCriterionDefinition.MobilityCriterionDefinitionBuilderImpl.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Those `@JsonInclude(JsonInclude.Include.NON_NULL)` are not really mandatory on the quality side. do you need that in gp?
In that case, maybe it would be better to add it in all CritetironDefinition classes

Reply:
We are assuming this is sometimes overriding defaults in the traveloptions and then core takes a different default (setting it to null could be differently interpreted than not setting it at all) - or something like. via LOOP it passes through many services and we just want to avoid that something is wrongly set to "null"

but we don't think this is the issue tbh. in any case it helps the readability of the resulting json :)

Reply:
we could also set it to the other ones. I was simply picking the ones where I see potential that it is used in the LOOP geo ai model

Reply:
let me know if I should add it to all of them

Reply:
Ok, I got it.
Can you please add it to all of them, then!